### PR TITLE
OSDOCS#8294: adding Agent Known Issue to RNs

### PR DESCRIPTION
**Pull request by skopacz1**
[OSDOCS-8294](https://issues.redhat.com/browse/OSDOCS-8294)

Version(s):
4.14 only

This PR adds a known issue to the 4.14 release notes regarding failed agent installs on vSphere clusters with multiple worker nodes.

QE review:
- [x] QE has approved this change.

preview: [Known Issues](https://66650--docspreview.netlify.app/openshift-enterprise/latest/release_notes/ocp-4-14-release-notes.html#ocp-4-14-asynchronous-errata-updates:~:text=Agent%2Dbased%20installations%20on%20vSphere%20with%20multiple%20worker%20nodes%20might%20fail%20due%20to%20a%20failure%20to%20remove%20node%20taint%2C%20which%20causes%20the%20installation%20to%20be%20stuck%20in%20a%20pending%20state.)

### CODE REVIEW

File: release_notes/ocp-4-14-release-notes.adoc [hunk 1]
```diff
@@ -2135,6 +2135,10 @@ To make the required change, follow the instructions in link:https://access.redh
 
 * Broadcom network interface controllers in legacy Single Root I/O Virtualization (SR-IOV) do not support quality of service (QoS) and tag protocol identifier (TPID) settings for the SRIOV VLAN. This affects Broadcom BCM57414, Broadcom BCM57508, and Broadcom BCM57504. (link:https://issues.redhat.com/browse/RHEL-9881[*RHEL-9881*])
 
+* Agent-based installations on vSphere with multiple worker nodes might fail due to a failure to remove node taint, which causes the installation to be stuck in a pending state.
```
Comment (rwsu) [inline release_notes/ocp-4-14-release-notes.adoc:2138]:
```suggestion
* Agent-based installations on vSphere will fail due to a failure to remove node taint, which causes the installation to be stuck in a pending state.
```

Comment (rwsu) [reply release_notes/ocp-4-14-release-notes.adoc:2138]: The issue also happens with compact clusters that have no workers.

**Comment by skopacz1** [comment]
@mhanss could you PTAL when you have a chance? Thanks!

File: release_notes/ocp-4-14-release-notes.adoc [hunk 2]
```diff
@@ -2135,6 +2135,10 @@ To make the required change, follow the instructions in link:https://access.redh
 
 * Broadcom network interface controllers in legacy Single Root I/O Virtualization (SR-IOV) do not support quality of service (QoS) and tag protocol identifier (TPID) settings for the SRIOV VLAN. This affects Broadcom BCM57414, Broadcom BCM57508, and Broadcom BCM57504. (link:https://issues.redhat.com/browse/RHEL-9881[*RHEL-9881*])
 
+* Agent-based installations on vSphere will fail due to a failure to remove node taint, which causes the installation to be stuck in a pending state.
```
Comment (mhanss) [inline release_notes/ocp-4-14-release-notes.adoc:2138]: Please add that the SNO cluster is not impacted.

**Review by ShaunaDiaz — APPROVED** [review]
/lgtm

File: release_notes/ocp-4-14-release-notes.adoc [hunk 3]
```diff
@@ -2154,6 +2154,11 @@ To make the required change, follow the instructions in link:https://access.redh
 
 * In hosted control planes for {product-title}, on the bare metal and {VirtProductName} platforms, the auto-repair function is disabled. (link:https://issues.redhat.com/browse/OCPBUGS-20028[*OCPBUGS-20028*])
 
+* Agent-based installations on vSphere will fail due to a failure to remove node taint, which causes the installation to be stuck in a pending state.
+{sno-caps} clusters are not impacted.
+There is no current workaround.
+(link:https://issues.redhat.com/browse/OCPBUGS-20049[*OCPBUGS-20049*])
```
Comment (jeana-redhat) [inline release_notes/ocp-4-14-release-notes.adoc:2160]: Did you intend these line breaks to render as new lines? If not, suggest removing the hard breaks.

**Review by jeana-redhat — COMMENTED** [review]
One question/consideration

/remove-label merge-review-in-progress
/remove-label merge-review-needed

File: release_notes/ocp-4-14-release-notes.adoc (hunk 3 above)
Comment (skopacz1) [reply release_notes/ocp-4-14-release-notes.adoc:2160]: These are all meant to be parts of a single paragraph, I'm just splitting them up in the adoc for a cleaner look/better ability to reference specific sentences.

**Pull request merged**